charts/ffperf/templates/statefulset.yaml

```diff
@@ -36,6 +36,9 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       labels:
+        {{- if .Values.podLabels }}
```

Comment:
Extra pod labels are useful when want to register pods to work within existing, genercized NetworkPolicies.